docs/deploy-cephfs.md

```diff
@@ -49,6 +49,7 @@ make image-cephcsi
 | `--domainlabels`          | _empty_                     | Kubernetes node labels to use as CSI domain labels for topology aware provisioning, should be a comma separated value (ex:= "failure-domain/region,failure-domain/zone")                                                                                                             |
 | `--enable-read-affinity` | `false`                       | enable read affinity                                                                                                                                                                                                                                                                 |
 | `--crush-location-labels`| _empty_                       | Kubernetes node labels that determine the CRUSH location the node belongs to, separated by ','.<br>`Note: These labels will be replaced if crush location labels are defined in the ceph-csi-config ConfigMap for the specific cluster.`                                                                                                                                                                                       |
+| `--radosnamespacecephfs`| `csi`                       | CephFS RadosNamespace used to store CSI specific objects and keys.                                                                                                                               |
```

Comment:
default is not `csi` in commandline argument,

Reply:
> default is not `csi` in commandline argument,

Internally it is the value `csi` if not set, so i should update docs with `_empty_` ?

Reply:
As this doc reflect the default in arguments lets keep it to `_empty_`

Reply:
Rebased and fixed.